public/app/features/connections/tabs/ConnectData/NoAccessModal/NoAccessModal.tsx

```diff
@@ -0,0 +1,117 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2, Modal, Icon, Button } from '@grafana/ui';
+
+import { type CardGridItem } from '../CardGrid';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  modal: css`
+    width: 500px;
+  `,
+  modalContent: css`
+    overflow: visible;
+    color: ${theme.colors.text.secondary};
+
+    a {
+      color: ${theme.colors.text.link};
+    }
+  `,
+  description: css`
+    margin-bottom: ${theme.spacing.gridSize * 2}px;
```

Comment:
I think the preferred approach is to use the `theme.spacing` function without the need to multiply or use `px`.
```suggestion
    margin-bottom: ${theme.spacing(2)}
```

Same goes for all the `theme.spacing.gridSize` usage here.

Reply:
Sorry, I have missed it, taking care of it in a separate PR: https://github.com/grafana/grafana/pull/61684